.env

```diff
@@ -3,10 +3,10 @@ VUE_APP_FRONTEND=localhost:8080
 
 VUE_APP_BACKEND_PLIOS=plios/
 VUE_APP_BACKEND_ITEMS=items/
+VUE_APP_BACKEND_VIDEOS=videos/
+VUE_APP_BACKEND_QUESTIONS=questions/
+
 
-VUE_APP_BACKEND_PLIOS_LIST=/plios_list
-VUE_APP_BACKEND_PLIO_DETAILS=/get_plio
```

Comment:
this should not be removed in this PR - will be removed in my PR because this is used for Player too.